src/components/ReservationModal.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import { WHITE, BLACK } from '../colors';
+import { Modal } from 'antd';
+
+interface ReservationModalProps {
+    readonly status: string;
+    readonly blockID: number;
+}
+
+const StyledModal = styled(Modal)`
+    height: 40px;
+    background: ${WHITE};
+    color: ${BLACK};
+    font-size: 18px;
+    font-weight: 400;
+`;
+
+function error(blockID: number) {
+    StyledModal.confirm({
+        title: 'Are you sure?',
+        content: 'You want to release block ' + blockID,
+      onOk() {
+        console.log('Release');
+      },
+      onCancel() {
+        console.log('Cancel');
+      },
+    });
+  }
+
+function warning(blockID: number) {
+    StyledModal.confirm({
+        title: 'Are you sure?',
+        content: 'You want to reserve block ' + blockID,
+      onOk() {
+        console.log('Reserve');
+      },
+      onCancel() {
+        console.log('Cancel');
+      },
+    });
+  }
+
+function info(blockID: number) {
+    StyledModal.warning({
+        title: 'Sorry',
+        content: 'Block ' + blockID + ' is not available to reserve/release',
+    });
+  }
+
```

Comment:
I think a good way to do this would be to create a function which returns the content the modal requires. 

```
const getModalCopy = (status, blockId) => {
  switch (status) {
    case OPEN
```
Have this function return an object containing the copy.

Then, we'll use that content below.